packages/components/pages/f-checkout/src/components/Checkout.vue

```diff
@@ -332,6 +346,16 @@ export default {
                 [EventNames.FormValid]: this.submitCheckout,
                 [EventNames.FormInvalid]: this.onInvalidCheckoutForm
             };
+        },
+
+        placeOrderNotes () {
+            const { courier, kitchen, order } = this.notes;
+            return {
+                NoteForRestaurant: order?.note || null,
+                ...(kitchen?.note && { NoteForKitchen: kitchen.note }),
+                ...(courier?.note && { NoteForDelivery: courier.note }),
+                ...(courier?.note && { NoteForDriver: courier.note })
```

Comment:
Unfortunately, yes we do need to add the same field twice under 2 different keys 🤦‍♂️

Reply:
You could reduce this down a little?

```js
...(courier?.note && { 
	NoteForDelivery: courier.note,
	NoteForDriver: courier.note
})
```

Reply:
Much better. Thanks @DamianMullins - it was a last minute requirement so I just slapped another property on.